.github/labeler.yml

```
@@ -2,18 +2,15 @@
   - "apps/site/**"
 
 "area: apps":
-  - "apps/**"
+  - ["apps/**", "!apps/site/src/_pages/**"]
```

I'm unsure about this [syntax](https://github.com/actions/labeler#usage), but needs this to be
```suggestion
  - all: ["apps/**", "!apps/site/src/_pages/**"]
```
or
```suggestion
  - "apps/**"
  - "!apps/site/src/_pages/**"
```
?
I think the latter is the one we want: Trigger in `apps/**` but does not trigger by `apps/site/src/_pages/**`

---

Let's try it